pkg/operator/ceph/object/objectstore.go

```diff
@@ -358,3 +361,75 @@ func GetObjectBucketProvisioner(c *clusterd.Context, namespace string) string {
 	}
 	return provName
 }
+
+func checkDashboardUser(context *Context) bool {
```

Comment:
Please add some unit tests for these new methods

Reply:
Done